easybuild/easyconfigs/t/tensorstore/tensorstore-0.1.72-gfbf-2024a.eb

```diff
@@ -0,0 +1,397 @@
+    'com_google_brotli',
+    'org_sourceware_bzip2',
+    'org_nghttp2',
+    'com_github_nlohmann_json',
+    'org_aomedia_avif',
+    'com_github_cares_cares',
+    'libwebp',
+    'nasm',
+    'com_google_snappy',
+    'tinyxml2',
+    'org_blosc_cblosc',
+    'blake3'
+]
+
+# Generate list of includ paths and link path for syslibs to be passed to Bazel as --copt and --linkopt
+_bzl_include_copts = ' '.join(['--copt=-I$EBROOT%s/include' % x for x in [
+    'PYBIND11',
+    'LIBTIFF',
+    'LIBJPEGMINTURBO',
+    'ZLIB',
+    'LIBPNG',
+    'LZ4',
+    'ZSTD',
+    'CURL',
+    'XZ',
+    'BZIP2',
+    'LIBAVIF',
+    'TINYXML2',
+    'CMINARES',
+    'SNAPPY',
+    'BLOSC',
+    'LIBWEBP',
+    'NLOHMANN_JSON',
+    'BLAKE3MINC',
+]])
+
+_bzl_linkopts = _bzl_include_copts.replace('copt', 'linkopt').replace('include', 'lib').replace('-I', '-L')
+
+
+# Download all bazel third_party artifacts by components and provide them in _dist_dir.
+_dist_dir = '%(builddir)s/dist/'
+_extract_cmd = 'mkdir -p ' + _dist_dir + ' && ln -s %s ' + _dist_dir
+default_easyblock = 'Binary'
+# templates for Bazel sources download:
+_sources_namedotversion = [{
+    'filename': '%(name)s.%(version)s.tar.gz',
+    'extract_cmd': _extract_cmd,
+}]
+_sources_nameminvversion = [{
+    'filename': '%(name)s-v%(version)s.tar.gz',
+    'extract_cmd': _extract_cmd,
+}]
+_sources_nameminversion = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'extract_cmd': _extract_cmd,
+}]
+_sources_nameminversionbz2 = [{
+    'filename': '%(name)s-%(version)s.tar.bz2',
+    'extract_cmd': _extract_cmd,
+}]
+_sources_nameminversionzip = [{
+    'filename': '%(name)s-%(version)s.zip',
+    'extract_cmd': _extract_cmd,
+}]
+#
+default_component_specs = {
+    'install_cmd': '#',
+    'source_urls': [
+        "https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/bazelbuild/%(name)s/archive/",
+        "https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/bazel-contrib/%(name)s/archive/",
+        "https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/bazel-contrib/%(name)s/releases/download/" +
+        "v%(version)s/",
+        "https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/%(name)s/%(name)s/archive/",
+        "https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/google/%(name)s/archive",
+        'https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/google/%(name)s/releases/download/' +
+        '%(version)s/',
+        'https://storage.googleapis.com/tensorstore-bazel-mirror/sourceforge.net/projects/%(name)s/files/%(name)s/' +
+        '%(version)s',
+        'https://github.com/bazelbuild/%(name)s/releases/download/%(version)s/',
+        'https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/awslabs/%(name)s/archive/',
+        'https://storage.googleapis.com/tensorstore-bazel-mirror/github.com/aws/%(name)s/archive/'
+    ],
+    'sources': _sources_nameminversion,
```

Comment:
As a workaround it was suggested to add
'runtest': None,
here.
Could you please do that and I'll rerun the test build.